dataprofiler/tests/profilers/test_numerical_options.py

```diff
@@ -0,0 +1,103 @@
+from dataprofiler.profilers.profiler_options import BooleanOption, NumericalOptions
+from dataprofiler.tests.profilers.test_base_column_options import TestBaseColumnOptions
+
+
+class TestNumericalOptions(TestBaseColumnOptions):
+    
+    option_class = NumericalOptions
+    keys = ["min", "max", "sum", "variance", "histogram_and_quantiles"]
+
+    def test_init(self, *mocks):
+        options = self.get_options()
+        for key in self.keys + ['is_numeric_stats_enabled', 'is_enabled']:
+            self.assertTrue(key in options.properties)
+    
+    def test_set_helper(self, *mocks):
+        super().test_set_helper(*mocks)
+        options = self.get_options()
+
+        # Enable and Disable Options
+        for key in self.keys:
+            skey = '{}.is_enabled'.format(key)
+            for enabled in [True, False]:
+                options._set_helper({skey:enabled}, '') 
+                self.assertEqual(enabled, options.properties[key].is_enabled)        
+
+    def test_set(self, *mocks):
+        super().test_set(*mocks)
+        options = self.get_options()
+        
+        # Enable and Disable Options
+        for key in self.keys:
+            skey = '{}.is_enabled'.format(key)
+            for enabled in [True, False]:
+                options.set({skey:enabled}) 
+                self.assertEqual(enabled, options.properties[key].is_enabled)        
+    
+    def test_validate_helper(self, *mocks):
+        super().test_validate_helper(*mocks)
+        options = self.get_options()
+        optpth = self.get_options_path()
+    
+        # Set BooleanOptions' is_enabled to a non-boolean value
+        for key in self.keys:
+            skey = '{}.is_enabled'.format(key)
+            expected_error = "{}.{}.is_enabled must be a Boolean.".format(optpth, key)
+            
+            options.set({skey: "Hello World"})
+            self.assertEqual([expected_error], options._validate_helper())
+            options.set({skey: True})
+
+        # Disable Sum and Enable Variance
+        options.set({"sum.is_enabled": False, "variance.is_enabled": True})
+        expected_error = "{}: The numeric stats must toggle on the sum if the variance is toggled on.".format(optpth)
+        self.assertEqual([expected_error], options._validate_helper()) 
```

Comment:
I'm going to approve as we can fix as tech debt later in a style cleanup, but make sure we limit or width to < 80